morpheus/stages/general/monitor_stage.py

```diff
@@ -206,7 +206,7 @@ def _ensure_progress_bar(self):
             self._progress = MorpheusTqdm(desc=self._description,
                                           smoothing=self._smoothing,
                                           dynamic_ncols=True,
-                                          unit=self._unit,
+                                          unit=" {}".format(self._unit),
```

Comment:
What about something like this?
```python
unit=(self._unit if self._unit.startswith(" ") else " {}".format(self._unit))
```
If the user prepended a space themselves, this wouldnt double add the space.

Reply:
Good catch, thanks